source/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dependencies": {
+    "azure-pipelines-task-lib": "^4.3.0",
+    "azure-pipelines-tool-lib": "^2.0.0-preview",
+    "jest-junit": "^15.0.0"
```

Comment:
Is `jest-junit` really a `dependencies` or should it be `devDependencies` instead ?